serveradmin/resources/templates/resources/index.html

```diff
@@ -39,7 +39,8 @@
                 <div class="form-group row input-controls">
                     <label for="per_page" class="col-sm-1 col-form-label">Per Page:</label>
                     <div class="col-md-4">
-                        <input class="form-control form-control-sm" id="per_page" type="number" value="{{ per_page }}" />
+                        <input class="form-control form-control-sm" id="per_page" name="per_page" type="number" value="{{ per_page }}" />
+                        <input type="hidden" id="page" name="page" value="1" />
```

Comment:
```suggestion
                        <input type="hidden" id="page" name="page" value="{{ page }}" />
```
so, when accidentally submitting the form manually, it won't bring you back to the first page.